python/pyhit/pyhit.py

```diff
@@ -115,14 +115,21 @@ def setComment(self, *args):
             param, text = args
 
         comment = self.__hitparamcomments.get(param, self.__hitblockcomment)
-        if comment is not None:
+        if (comment is not None) and (text is not None):
             comment.setText('# {}'.format(text))
 
-        elif (comment is None) and (param is None):
+        if (comment is not None) and (text is None):
+            if comment is self.__hitblockcomment:
+                self.__hitblockcomment.remove()
+                self.__hitblockcomment = None
+            else:
+                comment.remove()
+
+        elif (comment is None) and (param is None) and (text is not None):
```

Comment:
How does this function deal with inline vs block comments?  Does it preserve pre-existing comment type?  When adding a new comment (instead of modifying) - does it create inline or block?  This should, at a minimum be described in the func doc comment.  Hit allows both inline and block comments basically anywhere.

Reply:
It is handled, see the code comments on line 107. There is a function self.__reinitcomments that collects the block and param comments and populates the dict() that is used on 117 here to get the param or block comment.